src/lib/url/url.js

```diff
@@ -1,6 +1,9 @@
-const queryParams = ['include', 'filter', 'fields', 'sort'];
 import { isEmpty } from '../../utils';
 
+const entrypointPath = '/jsonapi';
```

Comment:
Should this be hardcoded? I'm not really clear why you're overwriting the "path" the user puts in- why not just request a domain?

Reply:
> Should this be hardcoded?

Probably not. I'll make it use the process env. Ideally, the JSON:API  spec would have a well-known URI and a spec provision about what an entrypoint should look like (tons of context here: https://github.com/json-api/json-api/issues/1288)

> I'm not really clear why you're overwriting the "path" the user puts in- why not just request a domain?

This PR disconnects the "landing URL" (previously "homeUrl") from the "entrypoint URL".

The landing URL is the URL that the user intends to explore, so if you're looking at some endpoint and you think "hmmm, I think I should use the explorer for this, you can just copy the URL from Postman or Chrome or whatever and paste it in. Then the explorer will load the same thing you were looking at.

OTOH, he "entrypoint URL" is the URL for a resource which a REST client or explorer can "enter" the API space and discover all the other resources by following links.

If we kept the landing URL and the entrypoint the same, then if you entered `http://drupal.test/jsonapi/node/article` there'd be no way to switch resource types to view `/node/page` since `/node/article` does not have links to it.

All that said... this is why I've been hesitant about the entrypoint links from the start. The `/jsonapi` URL is a Drupalism and most JSON:API servers will not have a resource like it.

Reply:
It makes a lot more sense with the example of `jsonapi/node/article` -being able to jump directly to that url. It's very cool that you can go to something like `http://drupal.test/jsonapi/node/page?fields[node--page]=drupal_internal__nid` immediately and the UI responds.